great_expectations/core/evaluation_parameters.py

```diff
@@ -219,11 +219,10 @@ def build_evaluation_parameters(
 
             # First, check to see whether an argument was supplied at runtime
             # If it was, use that one, but remove it from the stored config
-            if "$PARAMETER." + value["$PARAMETER"] in value:
-                evaluation_args[key] = evaluation_args[key][
-                    "$PARAMETER." + value["$PARAMETER"]
-                ]
-                del expectation_args[key]["$PARAMETER." + value["$PARAMETER"]]
+            param_key = f"$PARAMETER.{value['$PARAMETER']}"
+            if param_key in value:
+                evaluation_args[key] = evaluation_args[key][param_key]
+                del expectation_args[key][param_key]
```

Comment:
What is happening here? I'm having a hard time following. Is this in response to a particular Issue?